CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Documenter.jl changelog
 
+## Version `v0.27.0`
+
+* ![Enhancement][badge-enhancement] The JS dependencies have been updated to their respective latest versions.
+
+  - highlight.js has been updated to `v10.5.0`, which also brings various updates to the highlighting of Julia code. Due to the changes in highlight.js, code highlighting will not work on IE11. ([#1503][github-1503])
```

Comment:
(All of) my changes are not in the official release, so is this really true?

Reply:
From the changelog of highlight.js:
 - enh(julia) Update keyword lists for Julia 1.x (#2781) Fredrik Ekre

Reply:
Right but that is just https://github.com/highlightjs/highlight.js/pull/2781 and does not include any of the other stuff that apparently was not working on IE11.

Reply:
I think the word "various" isn't correct, but the IE11 stuff has nothing todo with your work, but refers to: https://github.com/highlightjs/highlight.js/releases/tag/10.0.0 :
> If your users have very old browsers, they may no longer be supported (no more IE11, etc.). (We're using ES2015 code now.)

Reply:
Could you suggest a better phrasing? I am not really familiar with the changes in highlight.js

Reply:
I see, this looks ok then. I was told some of the changes from my blogpost didn't work on some older browsers so I thought you were talking about that.

Looks like IE11 has just 1% of users nowadays so maybe not even worth mentioning?